test/suite/test_timestamp22.py

```diff
@@ -44,6 +44,7 @@ class test_timestamp22(wttest.WiredTigerTestCase):
     rand = suite_random.suite_random()
     oldest_ts = 0
     stable_ts = 0
+    last_durable = 0
```

Comment:
Instead of just fixing the test, can we also add an assert in Wiredtiger to crash in this case?

Reply:
Wiredtiger crashes during the connection close call in this case. To avoid the test from crashing, I added this change.